Gemfile

```diff
@@ -25,4 +26,5 @@ gem 'puma', '~> 3.0'
 gem 'rack-cors'
 gem 'rest-client'
 gem 'swagger_ui_engine'
+gem 'topological_inventory-api-client', :git => "https://github.com/mkanoor/topological_inventory-api-client", :branch => "master"
```

Comment:
Is there any way to not include a fork as part of the Gemfile?

Reply:
@syncrou This is temporary till the code evolves on the topological service and they create a gem and repo and we use a specific branch. @Fryguy didn't want me to own or build that gem it belongs to the topology service. This is just a place holder so we can get this tested